From b317038a01094136d06d4cb769b7755450b94f61 Mon Sep 17 00:00:00 2001
From: Florian Bruhin <me@the-compiler.org>
Date: Mon, 18 Sep 2023 18:03:21 +0200
Subject: [PATCH] eventfilter: Also enable workaround on wayland-egl

Will be fixed nicely on main, this is a more minimal fix
---
 qutebrowser/keyinput/eventfilter.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qutebrowser/keyinput/eventfilter.py b/qutebrowser/keyinput/eventfilter.py
index 306d4405b6d..ce9959bd4e3 100644
--- a/qutebrowser/keyinput/eventfilter.py
+++ b/qutebrowser/keyinput/eventfilter.py
@@ -86,7 +86,7 @@ def eventFilter(self, obj: Optional[QObject], event: Optional[QEvent]) -> bool:
 
         if (
             ev_type == QEvent.Type.DragEnter and
-            objects.qapp.platformName() == "wayland" and
+            objects.qapp.platformName() in ["wayland", "wayland-egl"] and
             qVersion() == "6.5.2"
         ):
             # WORKAROUND for https://bugreports.qt.io/browse/QTBUG-115757
